diff --git a/node_modules/expo/android/src/main/java/expo/modules/ExpoReactHostFactory.kt b/node_modules/expo/android/src/main/java/expo/modules/ExpoReactHostFactory.kt
index a1da771..daaafdc 100644
--- a/node_modules/expo/android/src/main/java/expo/modules/ExpoReactHostFactory.kt
+++ b/node_modules/expo/android/src/main/java/expo/modules/ExpoReactHostFactory.kt
@@ -19,6 +19,7 @@ import com.facebook.react.runtime.JSRuntimeFactory
 import com.facebook.react.runtime.ReactHostDelegate
 import com.facebook.react.runtime.ReactHostImpl
 import com.facebook.react.runtime.hermes.HermesInstance
+import com.facebook.react.fabric.ReactNativeConfig
 import java.lang.ref.WeakReference
 
 object ExpoReactHostFactory {
@@ -71,6 +72,8 @@ object ExpoReactHostFactory {
         handler.onReactInstanceException(useDeveloperSupport, error)
       }
     }
+
+    override fun getReactNativeConfig(): ReactNativeConfig = ReactNativeConfig.DEFAULT_CONFIG
   }
 
   @OptIn(UnstableReactNativeAPI::class)
